src/formatDate.js

```diff
@@ -50,7 +50,24 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const from = Array.from(fromFormat);
```

Comment:
You are only using this in for declaration. I think you should add the pop method here and rename it ;) 
```suggestion
  const oldSeparator = Array.from(fromFormat).pop();
```

Reply:
done :)